go/vt/topo/locks.go

```diff
@@ -66,10 +76,29 @@ func init() {
 	for _, cmd := range FlagBinaries {
 		servenv.OnParseFor(cmd, registerTopoLockFlags)
 	}
+
```

Comment:
Will each module that supports dynamic configuration have to (need to?) write such a block of code? Or will it (can it?) be delegated to a helper that can take some metadata and callback?

Reply:
you would need to write your own code to consume the update notifications (if you want to do something like "log when the value changes" or "only check at most N seconds for a new value otherwise use the previous value").

if you want to just always take the current value in the config file (and also always pay the potential cost of the RWMutex), you can just call `myValue.Get()`. 

i think if we tried to create a single helper for this, it would be extremely tricky to support every use case (or turn into a mess of spaghetti code as we add edge case after edge case 😂)

Reply:
I think we should always be calling `myValue.Get()` unless there is a very good reason to only poll periodically.